src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -230,7 +236,6 @@ public static IEnumerable<Option<CodeStyleOption<bool>>> GetCodeStyleOptions()
             yield return PreferConditionalDelegateCall;
             yield return PreferPatternMatchingOverAsWithNullCheck;
             yield return PreferPatternMatchingOverIsWithCastCheck;
-            yield return PreferBraces;
```

Comment:
doesn't this need to be added somewhere else?

Reply:
➡️ No, that's covered by `CreateOption`. This static method was only used for simplifying the serialization code, and the prefer braces option was added there explicitly when it was removed from this helper.